integration-tests/src/test/kotlin/org/stellar/anchor/platform/subtest/Sep31Tests.kt

```diff
@@ -239,6 +257,8 @@ class Sep31Tests(config: TestConfig, toml: TomlContent, jwt: String) {
     JSONAssert.assertEquals(expectedAfterPatch, json(afterPatch), LENIENT)
   }
 
+  @Test
+  @Order(50)
```

Comment:
nit: I don't think we need to order this test.

Reply:
Will try to see if this fails the tests.